Library/Homebrew/cask/installer.rb

```diff
@@ -564,8 +564,12 @@ def load_installed_caskfile!
     end
 
     def load_cask_from_source_api!
-      options = { git_head: @cask.tap_git_head, sha256: @cask.ruby_source_checksum["sha256"] }
-      cask_source = Homebrew::API::Cask.fetch_source(@cask.token, **options)
+      cask_source = Homebrew::API::Cask.fetch_source(
+        @cask.token,
+        path:     @cask.ruby_source_path || "Casks/#{@cask.token}.rb",
```

Comment:
```suggestion
        path:     @cask.ruby_source_path,
```

maybe worth moving the `"Casks/#{@cask.token}.rb"` into there instead?

Reply:
That makes it harder to reason about in my opinion.

I think we could actually remove this fallback value once the JSON has been regenerated with `ruby_source_path` included. Instead, we can then fall back to the `formulae.brew.sh` URL if `path` is `nil` for some reason.